platform/linux-generic/odp_stash.c

```diff
@@ -118,6 +122,8 @@ int odp_stash_capability(odp_stash_capability_t *capa, odp_stash_type_t type)
 	capa->max_stashes          = CONFIG_MAX_STASHES - CONFIG_INTERNAL_STASHES;
 	capa->max_num_obj          = MAX_RING_SIZE;
 	capa->max_obj_size         = sizeof(uint64_t);
+	capa->max_get_batch        = MAX_BATCH_SIZE;
+	capa->max_put_batch        = MAX_BATCH_SIZE;
```

Comment:
Matter of taste, but could use `MIN_RING_SIZE` directly here.

Reply:
Changed in V3.